packages/subgraph/src/kpi-tokens-manager.ts

```diff
@@ -183,3 +222,92 @@ export function handleUpgradeTemplate(event: UpgradeTemplateEvent): void {
     }
     newTemplate.save();
 }
+
+export function handleSetFeatureSetOwner(event: SetFeatureSetOwnerEvent): void {
+    const oraclesManager = getKPITokensManager();
+    const templateSet = getTemplateSet(
+        bytesToAddress(oraclesManager.id),
+        event.params.templateId,
+    );
+    templateSet.featuresOwner = event.params.owner;
+    templateSet.save();
+}
+
+export function handleEnableFeatureFor(event: EnableFeatureForEvent): void {
+    const feature = getTemplateFeature(
+        event.params.templateId,
+        event.params.featureId,
+    );
+
+    const allowedFeatureAccountId = feature.id.concat(
```

Comment:
Wondering if we should move this logic to a dedicated function in the `common.ts` file, since we are using this in multiple places.

Reply:
Addressed in dd931945acc44f5ce2a4e0a39d5590012448dff8